common/src/main/java/com/wynntils/models/containers/BankModel.java

```diff
@@ -157,12 +216,13 @@ public void toggleEditingName(boolean editingName) {
         this.editingName = editingName;
     }
 
-    private Storage<Map<Integer, String>> getCurrentNameMap() {
+    private Map<Integer, String> getCurrentNameMap() {
         return switch (currentContainer) {
-            case BANK -> customBankPageNames;
-            case BLOCK_BANK -> customBlockBankPageNames;
-            case BOOKSHELF -> customBookshelfPageNames;
-            case MISC_BUCKET -> customMiscBucketPageNames;
+            case ACCOUNT_BANK -> customBankPageNames.get();
+            case BLOCK_BANK -> customBlockBankPageNames.get();
+            case BOOKSHELF -> customBookshelfPageNames.get();
+            case CHARACTER_BANK -> customCharacterBankPagesNames.get().get(Models.Character.getId());
```

Comment:
perhaps getOrDefault?

Reply:
Also still missing getOrDefault

Reply:
https://github.com/Wynntils/Artemis/pull/2140/commits/daf3428fa886f2d561a11af5696a7aab6e641282 ?

Reply:
Seems like it got reverted?

Reply:
Oh wait no this is for `getCurrentNameMap` not `getFinalPage` where I added it